packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -126,6 +118,10 @@
   block-size: fit-content;
   line-height: var(--iui-size-l);
   padding-inline: 1px; // To account for the left and right edges of the 1px border
+  transition:
+    background-color var(--iui-duration-1) ease-out,
+    border-color var(--iui-duration-1) ease-out,
+    color var(--iui-duration-1) ease-out;
```

Comment:
Does the transition for `border-color` already come from the `::before`?

Reply:
You are correct, removed.